entities/cb_game.rb

```diff
@@ -0,0 +1,134 @@
+class CBGame
+  include CodebreakerCaptainjns
+  include DataUtils
+
+  def self.call(env)
+    new(env).response.finish
+  end
+
+  def initialize(env)
+    @request = Rack::Request.new(env)
+  end
+
+  def response
+    case @request.path
+    when '/' then main
+    when '/rules' then Rack::Response.new(render('rules.html.erb'))
+    when '/statistics' then Rack::Response.new(render('statistics.html.erb'))
+    when '/start' then start
+    when '/play_again' then play_again
+    when '/check' then check
+    when '/lose' then lose
+    when '/game' then game
+    when '/win' then win
+    when '/show_hint' then show_hint
+    else Rack::Response.new('Not Found', 404)
+    end
+  end
+
+  private
+
+  def main
+    return redirect('game') if @request.session.key?(:game)
+
+    Rack::Response.new(render('menu.html.erb'))
+  end
+
+  def play_again
+    @request.session.delete(:game)
+
+    redirect
+  end
+
+  def start
+    return redirect('game') if @request.session.key?(:game)
+
+    return redirect unless @request.params.key?('player_name')
+
+    @request.session.delete(:result)
+    @request.session[:hints] = []
+    @request.session[:save] = false
+    name = @request.params['player_name']
+    difficulty = @request.params['level']
+    @request.session[:game] = Game.new(name: name, difficulty: difficulty)
```

Comment:
You can use `slice`
`@request.session[:game] = Game.new(@request.params.slice(:name, :difficulty))`